### PR TITLE
add audio parameter

### PR DESCRIPTION
**Pull request by jpgonzales27**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/models/parameter/AudioParameter.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * <p>
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/models/parameter/AudioParameter.java]: this should be deleted, maybe it was a typo error

Comment (jpgonzales27) [reply src/main/java/org/fundacion/jala/converter/models/parameter/AudioParameter.java]: thanks dani, i will correct it

File: src/main/java/org/fundacion/jala/converter/models/parameter/AudioParameter.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    public AudioParameter(final String newFilePath, final String newFormat, final String newBitrate, final String newHz, final String newVolume, final String newAudioChannels) {
+        super(newFilePath);
+        this.format = newFormat;
+        this.bitrate = newBitrate;
+        this.hz = newHz;
+        this.volume = newVolume;
+        this.audioChannels = newAudioChannels;
+    }
+
+    /**
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/models/parameter/AudioParameter.java:42]: The description of all methods in this class is missing

File: src/main/java/org/fundacion/jala/converter/models/parameter/Parameter.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    private String filePath;
+
+    public Parameter(final String newFilePath) {
+        this.filePath = newFilePath;
+    }
+
+    public Parameter() {
+    }
+
+    /**
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/models/parameter/Parameter.java:28]: The description of all methods in this class is missing too

File: src/main/java/org/fundacion/jala/converter/models/parameter/AudioParameter.java (hunk 2 above)
Comment (jpgonzales27) [reply src/main/java/org/fundacion/jala/converter/models/parameter/AudioParameter.java:42]: ok, i will correct it

Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/models/parameter/AudioParameter.java:42]: yes, I think that is missing description in the methods

File: src/main/java/org/fundacion/jala/converter/models/parameter/Parameter.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * <p>
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/models/parameter/Parameter.java]: this  should be deleted, maybe it was a typo error

**Review by danielaSantaCruz — APPROVED** [review]
Great job!!

File: src/main/java/org/fundacion/jala/converter/models/parameter/AudioParameter.java (hunk 2 above)
Comment (jpgonzales27) [reply src/main/java/org/fundacion/jala/converter/models/parameter/AudioParameter.java:42]: I don't understand, what is missing

**Review by cristianfchq — APPROVED** [review]

**Pull request merged**